.github/workflows/build.yml

```diff
@@ -45,9 +45,69 @@ jobs:
     name: Deploy cookbook
     needs: [lint_and_unittest, build_kirby]
     runs-on: ubuntu-latest
+    env:
+      DOMAIN: ${{ github.ref == 'refs/heads/master' && 'kirby.design' || '650b277bd9a54e5cbadc.westeurope.aksapp.io' }}
     steps:
       - name: Checkout repo
         uses: actions/checkout@v2
+      - name: Get branch name
+        if: github.ref != 'refs/heads/master'
+        run: |
+          branchname="${GITHUB_REF##*/}"
+          echo "Github branchname: ${branchname}"
+          branchname="${branchname##*/}"
+          echo "Removed leading paths - branchname: ${branchname}"
+          branchname=$(tr [:upper:] [:lower:] <<< "$branchname")
+          echo "Removed uppercase - branchname: ${branchname}"
+          branchname="${branchname//[^a-zA-Z0-9]/-}"
+          echo "Removed illegal chars - branchname: ${branchname}"
+          branchname=$(tr -s '-' '-' <<< "$branchname")
+          echo "Removed duplicated dashes - branchname: ${branchname}"
+          branchname="${branchname%%-}"
+          echo "Removed trailing dash - final branchname: ${branchname}"
+          releasename="kby-${branchname}"
+          releasename=$(cut -c 1-53 <<< "$releasename")
+          echo "Releasename: ${releasename}"
```

Comment:
@bogedal I added the echo's as a way to document what's going on here - if you're not a bash shark (like I'm not 👶 ) then it was pretty hard to figure out what was going on (and therefore hard to spot why the branch name had the first character removed 🤔 )